test/tests.js

```diff
@@ -274,6 +298,13 @@ QUnit.test('with attributes', function (assert) {
 	assert.strictEqual(document.cookie, '', 'should delete the cookie');
 });
 
+QUnit.test('deletion with unofficial attributes', function (assert) {
+	assert.expect(1);
+	var attributes = { path: '/', unofficial: 'anything' };
```

Comment:
We need to make sure it's required, otherwise we will create code to support speculation and that's not good =(. We can either remove the test and leave it as an unsupported feature until we get a request or research how is the browser behavior and document that.

Since we are not "officially" supporting custom attributes I don't think we need to make the research. However, if we are going to document that we support it, then we definitely need to make sure what we are doing.